DSMRloggerAPI.ino

```diff
@@ -77,10 +81,10 @@ void displayStatus()
     case 4:   sprintf(cMsg, "IP %s", WiFi.localIP().toString().c_str());
               break;
     default:  sprintf(cMsg, "Telgrms:%6d/%3d", telegramCount, telegramErrors);
-              msgMode = 0;
+              break;
   }
   oled_Print_Msg(3, cMsg, 0);
-  msgMode++;
+  msgMode= (msgMode+1) % 5; //modular 5 = number of message displayed (hence it cycles thru the messages
```

Comment:
Dit zou, om dezelfde logica als voorheen te krijgen 

msgMode= (++msgMode) % 5;

Moeten worden. Het zal zó ook werken maar je verandert iets wat werkt en je weet niet of het ergens anders misschien wel een probleem gaat geven (zal in dit geval niet zo zijn, maat toch).